deploy/kiali/kiali_cr.yaml

```diff
@@ -908,7 +908,7 @@ spec:
 #
 #      If you wish to ignore one or more validation errors, put their validation codes (e.g. KIA0101) in this list.
 #      ---
-#      ignore: []
+#      ignore: ["KIA1201"]
```

Comment:
Now that we have the automated cr docs being generated, this file is deprecated (though we can still manage it for now if we want, so you can leave this line 911 alone).. but you now have to modify this file with the new default: https://github.com/kiali/kiali-operator/blob/master/crd-docs/cr/kiali.io_v1alpha1_kiali.yaml#L375-L377

I would change those lines 375-377 so they look like this:
```
    validations:
      ignore: ["KIA1201"]
```

Reply:
Thanks, resolved.